web/migrations/0054_discount.py

```
@@ -0,0 +1,32 @@
+from django.conf import settings
+from django.db import migrations, models
+
+import web.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("web", "0053_goods_featured"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)